server/grammar/Yml.g4

```diff
@@ -119,7 +119,7 @@ INTEGER: NUMBER+;
 // Colon are OK inside a YMLID
 // Colons aren't interpreted by YE in a particular way.
 YMLID: ID (COLON? COLON ID)*;
-ID: LETTER ALPHANUM*;
+ID: ALPHANUM* LETTER ALPHANUM*;
```

Comment:
Wut? IDs starting with numerals? Intentional? YML?

Reply:
Wep. It's completely valid. I added it on purpose.